h/assets.py

```diff
@@ -85,12 +85,16 @@ def urls(self, bundle):
         Returns the URLs at which all files in a bundle are served,
         read from the asset manifest.
         """
-        manifest = self.manifest.load()
         bundles = self.bundles.load()
 
-        def asset_url(path):
-            return '{}/{}'.format(self.assets_base_url, manifest[path])
-        return [asset_url(path) for path in bundles[bundle]]
+        return [self.asset_url(path) for path in bundles[bundle]]
+
+    def asset_url(self, path):
```

Comment:
This class is `h.assets.Environment`, so `asset_url` is possibly a bit redundant. The method above is called `urls`, so maybe call this one `url`?